docs/source/api/spi.rst

```diff
@@ -0,0 +1,20 @@
+###
+SPI
+###
+
+About
+-----
+
+For some APIs, the reference to be used is the same as the Arduino Core.
+
+Arduino API Reference
+---------------------
+
+`SPI Reference <https://www.arduino.cc/reference/en/language/functions/communication/spi/>`_
+
+`SPI Description <https://docs.arduino.cc/learn/communication/spi>`_
+
+Examples
+--------
+
```

Comment:
Missing the sentences "To get started with Hall sensor, you can try:"

There is now just one example,[SPI_Multiple_Buses](https://github.com/espressif/arduino-esp32/tree/master/libraries/SPI/examples/SPI_Multiple_Buses). Maybe it can be mentioned?